### PR TITLE
Please update Xuxiang's Codingbat Assignment

### PR DESCRIPTION
**Pull request by maoxuxiang**
The URL is here: http://maoxuxiang.github.io/spring2014/2014/02/09/Xuxiang-CodingBat.html
Thanks!

### CODE REVIEW

File: _posts/2014-02-09-Xuxiang-CodingBat.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+it seems very simply and seems only one main line code can solve this problem. But actually it used two Python Build-in function: one is: sum. another is : slice. this two function helped me a lot and got rid of many verbose code. I like it!
+
+
+# List-1 > make_ends:
+
+
+Given an array of ints, return a new array length 2 containing the first and last elements from the original array. The original array will be length 1 or more. 
+
+I write code like this:
+```
```
Comment (eah13) [inline _posts/2014-02-09-Xuxiang-CodingBat.md:49]: you need a blank line above the ```

Comment (eah13) [inline _posts/2014-02-09-Xuxiang-CodingBat.md:49]: screenshot 2014-02-09 at 2 49 49 pm

**Comment by eah13** [comment]
Great job discussing your code.  JUst fix that code block and you'll be ready to merge

**Comment by gerbal** [comment]
@maoxuxiang your images are not loading. It appears Baidu doesn't make stable image links. Try using [Imgur.com](http://imgur.com) instead.

**Comment by maoxuxiang** [comment]
I have changed the links of my images. I think it works. Here is the link: 
http://maoxuxiang.github.io/spring2014/2014/02/09/Xuxiang-CodingBat.html

**Comment by eah13** [comment]
:boom:  Nice work.

**Pull request merged**